src/store/products/products.hooks.ts

```diff
@@ -0,0 +1,44 @@
+import axios, { AxiosResponse } from "axios";
+
+import { useApiFetch, useApiMutate } from "../api.hooks";
+import {
+  Product,
+  ProductsFetchResponseData,
+  PRODUCTS_FETCH_SUCCESS,
+  ProductsAddParams,
+  ProductsAddResponseData,
+  PRODUCTS_ADD_SUCCESS
+} from "./products.types";
+import { getProducts } from "./products.selectors";
+
+export const useFetchProducts = () =>
+  useApiFetch({
+    requestFn: async () => {
+      const res: AxiosResponse<ProductsFetchResponseData> = await axios.get(
+        `${process.env.REACT_APP_API_URL}/products`
```

Comment:
a call to an environment variable set elsewhere, in the .env file, which provides the root of the API, rather than state it in the code itself. Is this so that in future it will be easier to change that perhaps?